### PR TITLE
Add notifications for travel

### LINKED ISSUE #2529: Send reminders for travel assignment action items

### LINKED ISSUE #2620: Travel notifications
- [x] Send acknowledgement when DocuSign envelope is ingested
- [x] Send CTA when TA is created
- [x] Send reminder 48hrs after travel assignment is created (same logic as DTs)
- [x] Send notification (email+nova) to primary contact when all TAs are completed
- [x] Show Nova notifications for action items (individually?)

### PR DESCRIPTION
**Pull request by kberzinch**
Closes #2529 
Closes #2620

### CODE REVIEW

File: app/Mail/Travel/AllTravelAssignmentsComplete.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        $this->travel = $travel;
+    }
+
+    /**
+     * Build the message.
+     */
+    public function build(): self
+    {
+        return $this->from('noreply@my.robojackets.org', 'RoboJackets')
+                    ->to($this->travel->primaryContact->gt_email, $this->travel->primaryContact->name)
```
Comment (JoshOldenburg) [inline app/Mail/Travel/AllTravelAssignmentsComplete.php:35]: Maybe cc treasurer@, or even officers@?

File: resources/views/mail/travel/assignmentcreated.blade.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
+Hi {{ $assignment->user->preferred_first_name }},
+
+@if($assignment->travel->tar_required)
+You have been assigned to {{ $assignment->travel->name }}. Please complete the following items as soon as possible so that we can book travel for you.
+
+Visit {{ route('docusign.travel') }} to submit a Travel Authority Request. Georgia Tech requires this form to be submitted for all official travel.
+
+Make a ${{ intval($assignment->travel->fee_amount) }} payment for the travel fee.
```
Comment (JoshOldenburg) [inline resources/views/mail/travel/assignmentcreated.blade.php:8]: This should probably move below the conditional.

**Review by JoshOldenburg — CHANGES_REQUESTED** [review]
Couple quick things, otherwise looks good.

File: resources/views/mail/travel/assignmentcreated.blade.php (hunk 2 above)
Comment (kberzinch) [reply resources/views/mail/travel/assignmentcreated.blade.php:8]: Do you mean inside the `@else` or after the `@endif`?

Comment (JoshOldenburg) [reply resources/views/mail/travel/assignmentcreated.blade.php:8]: After the `@endif`.

Comment (kberzinch) [reply resources/views/mail/travel/assignmentcreated.blade.php:8]: The `@else` mentions the travel fee already.

Comment (JoshOldenburg) [reply resources/views/mail/travel/assignmentcreated.blade.php:8]: Oops. Disregard

**Pull request merged**